### ISSUE TITLE
Dark Theme "ui-widget-content" background is white

### ISSUE THREAD

**Issue by clinkadink** [EdddieN/machinon-domoticz_theme#312]
Hi, well done and many thanks on this amazing theme. It certainly brings Domotics into the 2020's.

I noticed since upgrading from 1.9.5 to 1.10.2, with the Dark theme set, the background colour of the popups (when clicking edit/three dots) is white, instead of dark grey. I have changed this on line 45 of dark_theme.css:

```
.ui-corner-all,
.display .ui-widget-content,
.ui-widget-content {
    color: var(--main-text-color);
}
```

to this which fixed the problem.

```
.ui-corner-all,
.display .ui-widget-content,
.ui-widget-content {
    color: var(--main-text-color);
    background: var(--main-bg-color);
}
```

Again, keep up the great work. It is very appreciated :)

And have a great New Year! :)

Cheers
Chris

**Comment by EdddieN** [comment]
Hi Chris,

Many thanks for raising this, I did notice something similar on my Evohome devices too a while back but never got around to raise the issue. I hope @DewGew , @davidlb  or @landaisbenj  can pick it up soon into a new merge. I'm not programmer myself. 

Many thanks for your kind comments!

**Comment by davidlb** [comment]
Thanks for your report, I'll check it next week

**Comment by davidlb** [comment]
@clinkadink in fact, it have been fixed few months ago in the beta branch feed6a418569042cf8cf3c5206c0c34360aef660

So it might be time to release a new version :)
